docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -1 +1,98 @@
 = Glossary of terms related to {ProductName}
+:icons: font
+:source-highlighter: highlightjs
+
+**{ProductName}** +
+A platform to automate the process of building, testing, and releasing applications. {ProductName} offers enterprise-grade security and customizable feature sets.   
+
+**build pipeline customization** +
+The ability to update and manage build pipelines for each component in an application. 
+
+**cluster** +
+A Kubernetes deployment with nodes that run containerized applications and a control plane that manages the nodes.
+
+**commit** +
+A change to one or more files. In {ProductName}, commits that you make to a linked git repository move through the pipeline and automatically get published.
+
+**component** +
+An image that {ProductName} builds from source code in a repository. One or more components that run together form an application.
+
+**Conftest** +
+A utility for testing structured configuration data. Use Conftest to validate container information.
+
+**enterprise contract (EC)** +
+A set of release policies that you apply to your application snapshot. Use the EC to prevent releases that are not compliant with Supply-chain Levels for Software Artifacts (SLSA) guidelines. 
+
+
+**integration test** +
+A pipeline that you set up in GitHub. When you add components, it tests each one individually, and then tests the application as a whole.
+
+**IntegrationTestScenario** +
+A parameter that the integration service uses to test application components.
```

Comment:
A parameter doesn't _feel_ like the right word. An ITS is a kubernetes resource that defines a pipeline to run, I believe.